src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -131,7 +131,7 @@ public function load(array $configs, ContainerBuilder $container)
                     'macvim' => 'mvim://open?url=file://%%f&line=%%l',
                     'emacs' => 'emacs://open?url=file://%%f&line=%%l',
                     'sublime' => 'subl://open?url=file://%%f&line=%%l',
-                    'phpstorm' => 'phpstorm://open?url=file://%%f&line=%%l',
+                    'phpstorm' => 'phpstorm://open?file=%f&line=%l',
```

Comment:
I don't think you should make this modification at all.

> The current file link format is based on https://github.com/aik099/PhpStormProtocol, but PHPStorm supports the protocol natively using a different syntax (Since PHPStorm 8)

It appears this is only true for PHPStorm 8 on **MacOS**.  Windows and Linux versions of PHPStorm still require https://github.com/aik099/PhpStormProtocol (or https://github.com/sanduhrs/phpstorm-url-handler on Linux).  The [documentation](http://symfony.com/doc/current/reference/configuration/framework.html#ide) even references **PhpStormProtocol** as a requirements for using the 'phpstorm' value for 'ide'.

Perhaps, there should be a new 'phpstorm-mac' value in the code with the new format ('phpstorm://open?file=%f&line=%l') and the documentation should be changed to have a generic example such as '**myide**://open?url=file://%%f&line=%%l' to avoid confusion with phpstorm.

In my particular case, I would have had zero problems if I had just set the value of the 'ide' to 'phpstorm'.  My mistake was seeing 'phpstorm://open..." in the example for another editor and using that misleading example instead.